File: backend/src/main/java/bookmytrip/Entity/Hotel.java
package bookmytrip.Entity;

import javax.persistence.*;
import javax.validation.constraints.*;

import lombok.*;

@Entity
@Getter @Setter
@PrimaryKeyJoinColumn(name = "hotel_Id")
public class Hotel extends Entry {

	@NotNull
	@Min(1) @Max(5)
	@Column(nullable = false)
	private Integer stars;
		
	@NotNull
	@Column(nullable = false)
	private Boolean breakfastIncl;
}

File: frontend/src/app/entity/hotel.ts
import { Contact } from './contact';
import { Review } from './review';

export interface Hotel {
    id: number;
    name: string;
    contact: Contact;
    reviews: Review[];
    avrgRating: number;
    numOfReviews: number;

    stars: number;
    breakfastIncl: boolean;
}